flake.nix

```diff
@@ -0,0 +1,21 @@
+{
+  inputs = {
+
+    nixpkgs.url = "nixpkgs/nixos-unstable";
+
+    nixpkgsOld.url = "nixpkgs/18.03";
```

Comment:
I am wondering for how long this will be available. I've asked on the nixpkgs dev matrix.